src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,90 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
 See https://www.pantsbuild.org/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
 
+2.0.0b0 (9/15/2020)
+-------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Upgrade to Pex 2.1.16. (#10782)
+  `PR #10782 <https://github.com/pantsbuild/pants/pull/10782>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Sandbox Pex temporary files. (#10772)
+  `PR #10772 <https://github.com/pantsbuild/pants/pull/10772>`_
+
+* fs_util cat-proto has --child-dir arg (#10228)
+  `PR #10228 <https://github.com/pantsbuild/pants/pull/10228>`_
+
+* Add the ability to capture all known scope options in run tracker. (#10778)
+  `PR #10778 <https://github.com/pantsbuild/pants/pull/10778>`_
```

Comment:
Maybe refactoring, too. It's a little of a gray area, but generally I think of "new features" as new things that end users can benefit from.

Your typical user/org likely isn't using this option.